src/Microsoft.NET.Sdk.Functions.MSBuild/Tasks/ZipDeploymentStatus.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Utilities;
+using Microsoft.NET.Sdk.Functions.Http;
+using Microsoft.NET.Sdk.Functions.MSBuild.Properties;
+using Newtonsoft.Json;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.NET.Sdk.Functions.MSBuild.Tasks
+{
+    internal class ZipDeploymentStatus
+    {
+        private const int MaxMinutesToWait = 3;
```

Comment:
I like the units in all the constants, it makes it easy to double check all of the Timespan.FromX method calls.